packages/integrations/vercel/src/image/shared.ts

```diff
@@ -10,7 +10,7 @@ export function getDefaultImageConfig(astroImageConfig: AstroConfig['image']): V
 }
 
 export function isESMImportedImage(src: ImageMetadata | string): src is ImageMetadata {
```

Comment:
This should be removed in favor of `isESMImportedImage` exported from `astro/assets/utils`